pkg/kubelet/cm/helpers_linux.go

```diff
@@ -44,16 +44,17 @@ const (
 	SharesPerCPU  = 1024
 	MilliCPUToCPU = 1000
 
-	// 100000 is equivalent to 100usec
-	QuotaPeriod    = 100000
+	// 100000 microseconds is equivalent to 100ms
+	QuotaPeriod = 100000
+	// 1000 microseconds is equivalent to 1ms
 	MinQuotaPeriod = 1000
 )
 
 // MilliCPUToQuota converts milliCPU to CFS quota and period values.
```

Comment:
Consider to update the comment here to reflect that `MilliCPUToQuota` will return quota in microseconds

Reply:
Thanks a lot for the review! I've addressed all the comments (in the same commit with rebase); please let me know if something else is worth changing.